polaris/hub/client.py

```diff
@@ -797,114 +774,75 @@ def _upload_benchmark(
             benchmark_json["datasetArtifactId"] = benchmark.dataset.artifact_id
             benchmark_json["access"] = access
 
-            path_params = (
-                "/v1/benchmark" if artifact_type == ArtifactSubtype.STANDARD.value else "/v2/competition"
-            )
+            path_params = "/v1/benchmark"
             url = f"{path_params}/{benchmark.owner}/{benchmark.name}"
             response = self._base_request_to_hub(url=url, method="PUT", json=benchmark_json)
             response_data = response.json()
 
             progress_indicator.update_success_msg(
-                f"Your {artifact_type} benchmark has been successfully uploaded to the Hub. "
+                f"Your benchmark has been successfully uploaded to the Hub. "
                 f"View it here: {urljoin(self.settings.hub_url, url)}"
             )
             return response_data
 
     def get_competition(
-        self,
-        owner: str | HubOwner,
-        name: str,
-        verify_checksum: ChecksumStrategy = "verify_unless_zarr",
+        self, owner: str | HubOwner, name: str, zarr_root_path: str | None = None
     ) -> CompetitionSpecification:
         """Load a competition from the Polaris Hub.
 
         Args:
             owner: The owner of the competition. Can be either a user or organization from the Polaris Hub.
             name: The name of the competition.
-            verify_checksum: Whether to use the checksum to verify the integrity of the dataset.
+            zarr_root_path: An optional path that specifies the cache location of the competition's dataset
+                on your local filesystem.
```

Comment:
Ok, I think I see where some confusion is coming from. I test the competition flow via a simple python script that calls `.load_competition(...)`, and then I invoke this numerous times while creating predictions, tweaking things, etc.

Because I use a simple python script, it recreates the competition object each time, redirecting the `zarr_root_path` back to the `PolarisFS` one. 

Do you typically use a Jupyter notebook and run the `.cache()` call in a separate cell?

Reply:
> Do you typically use a Jupyter notebook and run the .cache() call in a separate cell?

Yes, indeed. I do think this is valuable, but I think implementing it properly would be a little more complex than this. I could see this easily be confusing to the user. 

Hence my recommendation to leave it out for the time being.